src/context.rs

```diff
@@ -1,30 +1,52 @@
 use clap::ArgMatches;
 use std::env;
+use std::fs;
 use std::path::PathBuf;
 
 pub struct Context<'a> {
     pub current_dir: PathBuf,
+    pub dir_files: Vec<PathBuf>,
     pub arguments: ArgMatches<'a>,
 }
 
 impl<'a> Context<'a> {
     pub fn new(arguments: ArgMatches) -> Context {
-        // TODO: Currently gets the physical directory. Get the logical directory.
         let current_dir = env::current_dir().expect("Unable to identify current directory.");
-
-        Context {
-            current_dir,
-            arguments,
-        }
+        Context::new_with_dir(arguments, current_dir)
     }
 
+    #[allow(dead_code)]
```

Comment:
maybe im missing something but I think you can remove this guy since your using it on `line:15`

Reply:
That's what I was thinking, but I was still getting warnings before adding the `allow` attribute. 😕